### DISCUSSION TITLE
x-select not working after latest update (1.1.2)

### DISCUSSION THREAD

**Discussion by sharathdn1** [wireui/wireui#262] [BUG] [answered]
https://user-images.githubusercontent.com/12691864/167586514-03fc4d36-f22c-4340-bf93-f9ba6de5e72d.mov

onclcik option are not selecting...
description is showing as undefined.. (not defined any description) 
code below for reference..
                        < x-select
                            label="Select Status"
                            placeholder="Select one status"
                            :options="[
                                ['name' => 'Pending',  'id' => 1],
                                ['name' => 'In Progress', 'id' => 2],
                                ['name' => 'Stuck',   'id' => 3],
                                ['name' => 'Done',    'id' => 4],
                            ]"
                            option-label="name"
                            option-value="id"
                            wire:model.defer="model"
                        />

**Comment by ph7jack** [comment]
Hello @sharathdn1, describe your project dependencies version
- livewire
- alpine
- laravel
- wireui

**Comment by labomatik** [comment, marked as answer]
Same issue on my side, switching from wireui 1.0 to 1.1.3 is breaking the select component
livewire: 2.10.15
alpine: 3.8
laravel: 9
wireui: 1.1.3

But i've updated alpine to 3.10 and all is working.
